src/static/js/client/sidebar-search.js

```diff
@@ -74,6 +74,8 @@ export const info = {
     stoppedScrollingTimeout: null,
 
     indexDownloadStatuses: Object.create(null),
+
+    currentValue: null,
```

Comment:
Can we move this up (still on its own i.e. one blank line above/below) above the `workerStatus:` block? This is a nitpick LOL but it's more to do with DOM interaction than like, timeouts or abstract values as kept further below.